core/src/main/scala/org/http4s/DecodeResult.scala

```diff
@@ -30,9 +30,15 @@ object DecodeResult {
   def success[F[_], A](a: A)(implicit F: Applicative[F]): DecodeResult[F, A] =
     EitherT(F.pure(Right(a)))
 
+  def successT[F[_], A](a: A)(implicit F: Applicative[F]): DecodeResult[F, A] =
```

Comment:
See #4268.  If we proceed, we should start a deprecation cycle in 0.21 and move these down there.